### PR TITLE
working shutdown command

### PR DESCRIPTION
**Pull request by rminnich**
I've tested this in qemu and all variations work.

how to test is not that clear, so I've removed the test.
    Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

**Comment by rminnich** [comment]
dep is just unreal. If this builds it's a miracle.

**Comment by hugelgupf** [comment]
You've got something messed up with these commits. Also update dep to the newest version, and redo that?

**Comment by rminnich** [comment]
this is not needed, at least in this form. I missed that shutdown had been committed

File: cmds/shutdown/shutdown.go [hunk 1]
```diff
@@ -2,62 +2,50 @@
...
-		"suspend": unix.LINUX_REBOOT_CMD_SW_SUSPEND,
-	}
-)
+var opcodes = map[string]uint{
+	"halt":    unix.LINUX_REBOOT_CMD_POWER_OFF,
+	"-h":      unix.LINUX_REBOOT_CMD_POWER_OFF,
+	"reboot":  unix.LINUX_REBOOT_CMD_RESTART,
+	"-r":      unix.LINUX_REBOOT_CMD_RESTART,
+	"suspend": unix.LINUX_REBOOT_CMD_SW_SUSPEND,
+	"-s": unix.LINUX_REBOOT_CMD_SW_SUSPEND,
```
Comment (hugelgupf) [inline cmds/shutdown/shutdown.go]: gofmt?

**Pull request merged**